### ISSUE TITLE
PyTorch visualization fails with torch.jit.script, but works with torch.jit.trace

### ISSUE THREAD

**Issue by Jeremalloch** [lutzroeder/netron#837]
- Netron app and version: 5.3.1 (as pip package)
- OS and browser version: macOS 11.4

I have a relatively basic CNN (conv2d, concat, add, batch norm and relu) in PyTorch.  When I trace the model with torch.jit.trace Netron visualizes the model, but with torch.jit.script Netron is unable to visualize it (with error "Error loading Python module. Unsupported function 'torch.conv2d' ").  I know in a [previous issue](https://github.com/lutzroeder/netron/issues/547#issuecomment-735283680) this was an issue because the nn.Upsample layer was dependent on the input dimensions, but I'm curious why this is an issue for a model with more basic ops not dependent on input shape.

![Visualization of model](https://user-images.githubusercontent.com/16086901/139749809-5bc88282-3e77-4188-8d0d-58e10f2fc8eb.png)

I've attached a python script to torch.jit.script / torch.jit.trace the model and launch a netron server:

Please attach or link model files to reproduce the issue if necessary
```
import argparse
import copy
import tempfile

import netron
import torch
from torch import Tensor, nn

class ConcatTest1(nn.Module):
    """
    Test case involving concat & add
                x
              / | \
             /  |  \
            /   |   \
           /    |    \
          /     |     \
         /      |      \
        /       |       \
    conv1     conv2     conv3
      |         |         |
     bn1       bn2       bn3
      |         |         |
    relu1     relu2    relu3
        \     /         /
         \   /         /
         concat       /
           |         /
            \       /
             \     /
              \   /
               add
                |
              conv4
                |
               bn4
                |
              relu4
    """

    def __init__(self) -> None:
        super().__init__()
        self.conv1 = nn.Conv2d(8, 8, (3, 3), padding="same")
        self.bn1 = nn.BatchNorm2d(8)
        self.relu1 = nn.ReLU()

        self.conv2 = nn.Conv2d(8, 16, (3, 3), padding="same")
        self.bn2 = nn.BatchNorm2d(16)
        self.relu2 = nn.ReLU()

        self.conv3 = nn.Conv2d(8, 24, (3, 3), padding="same")
        self.bn3 = nn.BatchNorm2d(24)
        self.relu3 = nn.ReLU()

        self.conv4 = nn.Conv2d(24, 8, (3, 3), padding="same")
        self.bn4 = nn.BatchNorm2d(8)
        self.relu4 = nn.ReLU()

    def forward(self, x: Tensor) -> Tensor:
        y = self.conv1(x)
        y = self.bn1(y)
        y = self.relu1(y)

        z = self.conv2(x)
        z = self.bn2(z)
        z = self.relu2(z)

        y = torch.cat([y, z], dim=1)

        z = self.conv3(x)
        z = self.bn3(z)
        z = self.relu3(z)

        y = torch.add(y, z)

        y = self.conv4(y)
        y = self.bn4(y)
        y = self.relu4(y)

        return y


def view_netron(model: torch.nn.Module, 
                sample_input: torch.Tensor = None, 
                port: int = 8080) -> None:
    model = copy.deepcopy(model)
    if sample_input is not None:
        scripted_model = torch.jit.trace(model, sample_input)
    else:
        scripted_model = torch.jit.script(model)

    with tempfile.NamedTemporaryFile(
        suffix=".pt", prefix="view_netron_", dir="/tmp/", delete=False
    ) as fp:
        scripted_model.save(fp.name)
        netron.start(fp.name, address=port, browse=False)

if __name__ == "__main__":
    USE_JIT_TRACE = False
    model_torch = ConcatTest1()
    sample_input = torch.randn(1,8,32,32)
    if USE_JIT_TRACE:
        view_netron(model_torch, sample_input, port=8083)
    else:
        view_netron(model_torch, port=8083)

```
